drivers/sensor/realtek/rts5912/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2025, Realtek, SIBG-SD7
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+
+zephyr_library_sources_ifdef(CONFIG_TACH_RTS5912 rts5912.c)
```

Comment:
This could lead to an empty library. Please add the `_ifdef` to `drivers/sensor/realtek/CMakeLists.txt` so you have `add_subdirectory_ifdef(CONFIG_TACH_RTS5912 rts5912)`